db/migrate/20211123144714_add_recent_searches.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddRecentSearches < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_options, :oldest_search_log_date, :datetime
+
+    add_index :search_logs, [:user_id, :created_at]
```

Comment:
I'm not 100% sure but I don't think we need to index rows where `user_id` is null.

Reply:
Yes, this make sense, frontend will only use rows where `user_id` is set.